bindings/pyroot/pythonizations/CMakeLists.txt

```diff
@@ -123,11 +124,23 @@ foreach(val RANGE ${how_many_pythons})
     install(CODE "execute_process(COMMAND ${python_executable} -O -m py_compile ${localruntimedir}/${py_source})")
   endforeach()
 
+  # Create meta-target PyROOT2 and PyROOT3 (INTERFACE library)
+  # Export of targets are not supported for custom targets(add_custom_targets())
+  add_library(PyROOT${python_major_version_string} INTERFACE)
+  target_link_libraries(PyROOT${python_major_version_string} INTERFACE cppyy_backend${python_under_version_string} cppyy${python_under_version_string} ROOTPythonizations${python_under_version_string})
```

Comment:
Do we need anything for the Python modules of pythonizations, cppyy and cppyy_backend? Or just doing the liason with the target libraries is enough?

Reply:
liason with the target libraries should be enough, it should pull other dependencies.